### PR TITLE
Hearings Google Analytics

### PR DESCRIPTION
**Pull request by sharonwarner**
Connects #3626 

screen shot 2017-11-01 at 11 17 33 am

screen shot 2017-11-01 at 11 17 59 am

### CODE REVIEW

File: client/app/hearings/actions/Issue.js [hunk 1]
```diff
@@ -124,7 +179,13 @@ export const saveIssues = (worksheetIssues) => (dispatch) => {
...
         then(() => {
           dispatch({ type: Constants.SET_ISSUE_EDITED_FLAG_TO_FALSE,
-            payload: { issueId: issue.id } });
+            payload: { issueId: issue.id },
+            meta: {
+              analytics: {
+                category: CATEGORIES.HEARING_WORKSHEET_PAGE,
+                action: ACTIONS.EDIT_ISSUE
+              }
+            } });
```
Comment (nickheiner-usds) [inline client/app/hearings/actions/Issue.js]: Can you put the `});` on the next line?

**Comment by sharonwarner** [comment]
screen shot 2017-11-01 at 11 43 19 am

New actions are showing up in GA. I also updated the constants to be in all caps so the style matches up.

File: client/app/hearings/actions/Issue.js [hunk 2]
```diff
@@ -124,7 +178,14 @@ export const saveIssues = (worksheetIssues) => (dispatch) => {
         worksheet_issues_attributes: [issue] } } }).
         then(() => {
           dispatch({ type: Constants.SET_ISSUE_EDITED_FLAG_TO_FALSE,
-            payload: { issueId: issue.id } });
+            payload: { issueId: issue.id },
+            meta: {
+              analytics: {
+                category: CATEGORIES.HEARING_WORKSHEET_PAGE,
+                action: ACTIONS.EDIT_ISSUE
```
Comment (nickheiner-usds) [inline client/app/hearings/actions/Issue.js]: I am not totally sure why we aren't just using the default value of `Constants.SET_ISSUE_EDITED_FLAG_TO_FALSE` but I trust that you thought about it and this makes sense. :smile:

File: client/test/karma/hearings/actions/Dockets-test.js [hunk 3]
```diff
@@ -14,6 +15,12 @@ describe('.setNotes', () => {
         hearingIndex,
         notes,
         date
+      },
+      meta: {
+        analytics: {
+          category: CATEGORIES.DAILY_DOCKET_PAGE,
+          debounceMs: 1000
```
Comment (nickheiner-usds) [inline client/test/karma/hearings/actions/Dockets-test.js]: Let's extract `debounceMs` here too. (And we should probably share it in the `analytics.js` constants file.)

File: client/app/hearings/actions/Issue.js (hunk 2 above)
Comment (sharonwarner) [reply client/app/hearings/actions/Issue.js]: Yeah! I think edit-issue is clearer than set-issue-edited-flag-to-false to somebody unfamiliar with the codebase.

File: client/test/karma/hearings/actions/Dockets-test.js (hunk 3 above)
Comment (sharonwarner) [reply client/test/karma/hearings/actions/Dockets-test.js]: 👍

**Comment by sharonwarner** [comment]
Agreed with timing HTTP requests. I'm creating a new ticket for that.

**Pull request merged**